src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -4933,17 +4934,23 @@ public void enableNode() {
 
     @Override
     public void disableKeyspaceCreation() throws IOException {
-        LockKeyspaceUtils.lockKeyspaceCreation();
+        LockKeyspaceCreationSetting.instance.setTrue();
     }
 
     @Override
     public void enableKeyspaceCreation() {
-        LockKeyspaceUtils.unlockKeyspaceCreation();
+        LockKeyspaceCreationSetting.instance.setFalse();
     }
 
     @Override
     public boolean isKeyspaceCreationEnabled() {
-        return !LockKeyspaceUtils.isKeyspaceCreationLocked();
+        return !LockKeyspaceCreationSetting.instance.isTrue();
+    }
+
+    @Override
+    public void persistentEnableClientInterfaces() {
+        DisableClientInterfaceSetting.instance.setFalse();
+        instance.startTransports();
```

Comment:
If we can only persist the disabled setting on startup, how will we disable one of the DCs during the migration in case of:
* forward: turning off the pcloud interfaces so all requests solely hit rubix
* rollback: turning off the rubix interfaces so all requests solely hit pcloud

Currently we hit the sidecar's `disableClientInterfaces` endpoint which simply calls `stopRpcServer()` and `stopNativeTransport`

Reply:
ah good point. okay!